modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1253,6 +1253,10 @@ CV__DNN_INLINE_NS_BEGIN
       */
      class CV_EXPORTS_W_SIMPLE DetectionModel : public Model
      {
+
+     private:
+         bool nmsAcrossClasses = false;
```

Comment:
Please add the changes from https://github.com/dkurt/opencv/pull/1 so this private field may go to internal implementation.

Reply:
Did this, I added the commit from that PR to this one.